packages/react-components/react-search/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -1,17 +1,122 @@
+import * as React from 'react';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type SearchBoxSlots = {
-  root: Slot<'div'>;
+  /**
```

Comment:
I'd like other people to weigh in, but my initial thought here is to have `SearchBoxSlots` and `SearchBoxProps` extend `InputSlots` and `InputProps` directly, so you don't need to duplicate the docs and props here (and also so if we e.g. add another prop, it only needs to be added to Input).